lib/datura/file_types/file_tei.rb

```diff
@@ -11,7 +11,6 @@ class FileTei < FileType
 
   def initialize(file_location, options)
     super(file_location, options)
-    # TODO these paths no longer work now that override xsl is not inside of the data repo, fix during xslt sweep
```

Comment:
Is this an out of date description? Overrides _are_ in the collection repo, yes?

Reply:
Yeah, I think that might have been leftover from a phase of the rewrite to make this a gem where I thought that we were going to have an XSL scripts repo external to this....really glad that didn't end up being the case!!!